.changes/next-release/bugfix-AWSSDKforJavav2-1054797.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "bugfix",
+    "category": "AWS SDK for Java v2",
+    "contributor": "",
+    "description": "When parsing proxy port from system properties, exceptions will be ignored and logged as warning."
```

Comment:
This can be confusing and might appear we are ignoring the exception with this change.
Can we just mention saying 
"Changing logging level from Error to Warn when there are exceptions in parsing proxy config from System/Environment variables, as we ignore these exceptions and proceed."